azurerm/resource_arm_iothub.go

```diff
@@ -248,6 +248,12 @@ func resourceArmIotHub() *schema.Resource {
 				},
 			},
 
+			"enable_fallback_route": {
```

Comment:
Given that the API takes more values should mirror it and possible include them? 
```
"fallback_route": {
	Type:     schema.TypeList,
	MaxItems: 1,
	Optional: true,
	Elem: &schema.Resource{
		Schema: map[string]*schema.Schema{
			"enabled": {
				Type:     schema.TypeBool,
				Required: true,
			},
...
```
and the rest

Reply:
That's indeed better. I modified the code for this.